spotlight-client/src/contentApi/retrieveContent.test.ts

```diff
@@ -0,0 +1,24 @@
+// Recidiviz - a data platform for criminal justice reform
+// Copyright (C) 2020 Recidiviz, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+// =============================================================================
+
+import retrieveContent from "./retrieveContent";
+import US_ND from "./sources/us_nd";
+
+test("returns content for the specified tenant", () => {
```

Comment:
Will this test fail if the US_ND content is "malformed," e.g. references a key that doesn't map to an actual metric id? If not, I'd love to have some basic unit test that loops through tenants and ensures nothing is malformed.

Reply:
this test will not fail in that case. However, you will get a compile-time error on the malformed content object because Typescript will reject any key in that object that isn't part of the `MetricTypeId` union type. 

If the data were coming from an external source then that would be a different story, but since it isn't, I thought it would be reasonable to just rely on the static analysis for this?

Reply:
I buy that, if the compile-time error will be caught in the CI we've set up for this project? If it's possible to get a PR merged here without having passed through that check, then I'd request a unit test be created so we don't have bug-fix PRs for accidental misconfigurations. But if it's impossible to get to a merged PR with a misconfiguration, then we're good!

Reply:
@jessex see latest commit that failed (although I am a bit peeved that our pre-commit hooks are still not working; it shouldn't have even been possible for me to commit that change!)

Reply:
Sold! Thanks for proving that, @macfarlandian